libraryBuilds/com.google.code.gson-gson/com.google.code.gson-gson/build.gradle

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2015 the authors of j2objc-common-libs-e2e-test
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+dependencies {
+    j2objcTranslation 'com.google.code.gson:gson:2.3.1:sources'
+}
+
+j2objcConfig {
+    // package-info.java exists in multiple packages.
+    filenameCollisionCheck false
```

Comment:
- prefixes.properties doesn't help with filenames.
- no the library can be used just fine.
- package-info.java contains no classes, but is valid way of documenting your packages and quite common (for people who actually use javadoc).

Btw this file was just moved from one directory up - you reviewed this in the last PR.